scripts/setup-kmsv2-kind-cluster.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
This yaml is similar to https://github.com/Azure/kubernetes-kms/blob/master/scripts/setup-kind-cluster.sh except for the `hostPath` and `containerPath`s. Could we consolidate this to one and just generate yamls based on env var to distinguish between v1 and v2 test env?